templates/api_results.html

```diff
@@ -9,6 +9,13 @@ <h1>Movie Title: {{ movie_info.title }}</h1>
     <img src="{{ movie_info.poster }}">
 </p>
 
+<h1>Wikipedia Results</h1>
+<p>
+    <a href = "https://www.wikipedia.com">
```

Comment:
Recommend some text as well as an image so it's obvious to the user that there's a clickable link here. You can also add an alt text attribute https://www.w3schools.com/tags/att_img_alt.asp

Reply:
Text that instructs the user to click the image? Currently, this change has an image displayed but no plain text saying to click the Wikipedia logo.